packages/storybook/stories/Color.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import type { Meta } from '@storybook/react';
+import { StoryObj } from '@storybook/react';
+
+import config from '../tailwind.config';
+
+const { theme: { colors } } = config;
+// Raw colors are not meant to be used for styling
+delete colors.raw;
+// Current theme colors, here for legacy reasons
+delete colors.theme;
+// Current overlay colors, here for legacy reasons
+const { base, dark } = colors.overlay;
+colors.overlay = { base, dark };
+
+const flattenColorArr = (data: any, prefix: string): string[] => {
+  return Object.keys(data).reduce((acc: string[], el) => {
+    const item = data[el];
+    if (typeof item === 'string') {
+      acc.push(prefix + el);
+      return acc;
+    }
+    return acc.concat(flattenColorArr(item, prefix + el + '-'));
+  }, []);
+};
+const flatColorArr = flattenColorArr(colors, 'bg-');
+
+const meta: Meta = {
+  title: 'Color',
+  args: {
+    className: 'bg-white',
+  },
+  argTypes: {
+    className: {
+      control: 'select',
+      options: flatColorArr,
+      description: 'Color classnames',
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof HTMLDivElement>;
+
+export const Interactive: Story = {
+  render: ({ ...props }) => {
+    return <div {...props}>{props.className}</div>;
+  },
+  name: 'Interactive',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/C7n8EiXBwV1sYIEHkQHS8R/daily.dev---Design-System?type=design&node-id=450-2&mode=dev',
+    },
+  },
+};
+
+const RenderColor = ({ color, prefix, value }: {
+  color: any;
+  prefix: string;
+  value: string
+}) => {
+  if (typeof color === 'string') return <div
```

Comment:
I know prettier doesn't work here at the moment, but would it be worth just manually aligning some parts of the code?

Reply:
Not sure I'm following correctly, aligning which parts exactly?

Reply:
<img width="650" alt="Screenshot 2024-03-04 at 11 11 09 AM" src="https://github.com/dailydotdev/apps/assets/36197304/27b5f1a0-dbee-450f-9f47-1a48d0e48a45">
Well the ones below this comment, like props in new lines, try to organise the code. But don't worry too much about it.

Reply:
Oh right, I did a manual prettier, hope that helps?